scrapers/MetArtNetwork.yml

```diff
@@ -0,0 +1,42 @@
+name: "MetArtNetwork"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metartnetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="pull-left custom-photo-movie-description"]
+      $credits: //div[@class="pull-left custom-credits-left"]
```

Comment:
in some scenes the credits tab is missing so we miss the performers and the date
the same with info and title
sample urls 
`https://www.metartnetwork.com/model/monika-dee/movie/20200623/ANCHORED/`
`https://www.metartnetwork.com/model/zelda-b/movie/20200607/SEXY_LUMBERJACK/`

changing the fields to the following seems to work 

```yaml
Title: //h3/a/@title
```
```yaml
      Date:
        selector: //li[@class="navbar-custom-login"]/a/@href
        replace: 
          - regex: .+galleryDate=(\d{8})&.+
            with: $1
        parseDate: "20060102"
```
```yaml
      Performers:
        Name: //a/span[contains(.,"Cast:")]/../text()
```